src/base-app.js

```diff
@@ -49,6 +50,9 @@ class FusionApp {
     enhancers.push(enhancer);
     this.registered.set(getTokenRef(token), {value, aliases, enhancers});
   }
+  cleanup() {
+    return Promise.all(this.cleanups.map(fn => fn()));
```

Comment:
There should be also a way to query the status of an app, similar to `server.listening`

Reply:
If we end up needing that, it would be easy enough to add. I want to keep things as simple as possible for now though